### PR TITLE
Fix doodstream

### PR DESCRIPTION
**Pull request by alpha7504**

### CODE REVIEW

File: script.module.resolveurl/lib/resolveurl/plugins/doodstream.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class DoodStreamResolver(ResolveUrl):
     pattern = r'(?://|\.)(dood(?:stream)?\.(?:com?|watch|to|s[ho]|cx|la|w[sf]|pm|re))/(?:d|e)/([0-9a-zA-Z]+)'
 
     def get_media_url(self, host, media_id):
-        if host.endswith('.cx'):
+        if host.endswith('.wf'):
```
Comment (Gujal00) [inline script.module.resolveurl/lib/resolveurl/plugins/doodstream.py:35]: you need to add checking for `.wf` and not just replace `.cx` as that is still required
`if any(host.endswith(x) for x in ['.cx', '.wf']):` will do the job

**Review by Gujal00 — CHANGES_REQUESTED** [review]

**Comment by alpha7504** [comment]
Thanks for pointing that out.

**Comment by bleak-lodge** [comment]
Just a cosmetic note here, that `endswith()` also takes a tuple, don't have to use `any()` ;)

**Comment by bleak-lodge** [comment]
value can be a tuple, so `host.endswith(('.cx', '.wf'))` should do the same thing as with any()
https://docs.python.org/3/library/stdtypes.html#string-methods

**Comment by alpha7504** [comment]
Should I make this change? or leave it be

**Pull request merged**